fab/fabfile.py

```diff
@@ -1407,5 +1407,23 @@ def _execute_with_timing(fn, *args, **kwargs):
             timing_log.write('{}: {}\n'.format(fn.__name__, duration.seconds))
 
 
+def _get_github():
+    try:
+        from .config import GITHUB_APIKEY
+    except ImportError:
+        print 'Try creating a fab/config.py with your GitHub API key to automate this step.'
+        print 'See fab/config.example.py.'
+        username = raw_input('Github username: ')
+        password = getpass('Github password: ')
+        return login(
+            username=username,
+            password=password,
+        )
+    else:
+        return login(
+            token=GITHUB_APIKEY,
+        )
```

Comment:
👍 for now i think it'll just be the captain user and in a later stage we can specify who deployed what. usually the deployer has little knowledge of what they're actually deploying. they were just assigned to do it, so i don't think we lose that much information. adding a username though is definitely in the pipeline of things to do